src/inmanta/data/__init__.py

```diff
@@ -5442,6 +5442,22 @@ class EnvironmentMetricsTimer(BaseDocument):
     __primary_key__ = ("environment", "metric_name", "timestamp")
 
 
+class DiscoveredResources(BaseDocument):
+    """
+    discovered resources
+
+    :param environment: the environment of those resources
+    :param discovered_resource_name: The name of the resources
+    :param values: The values
+    """
+
+    environment: uuid.UUID
+    discovered_resource_name: str
+    values: list[object]
```

Comment:
Not sure if storing them all in a single row is the best approach here, something to think about in the actual design.